planet.rb

```diff
@@ -0,0 +1,16 @@
+class Planet
+  def initialize(name, color, mass_kg, distance_from_sun_km, fun_fact)
+    @name = name.capitalize
+    @color = color
+    raise ArgumentError, 'Mass must be a positive numeric value' unless mass_kg.to_i > 0
+    @mass_kg = mass_kg
+    raise ArgumentError, 'Distance must be a positive numeric value' unless distance_from_sun_km.to_i > 0
+    @distance_from_sun_km = distance_from_sun_km
+    @fun_fact = fun_fact
+  end
+  attr_reader :name, :color, :mass_kg, :distance_from_sun_km, :fun_fact
```

Comment:
Minor comment:  The `attr_reader` should be above `initialize` and below the `class Planet`